### DISCUSSION TITLE
Is this plugin dbal compatible?

### DISCUSSION THREAD

**Discussion by mirai-andrian** [andrewdwallo/filament-companies#67] [Q&A] [answered]
Is this plugin [dbal/](https://filamentphp.com/docs/2.x/admin/resources/getting-started#automatically-generating-forms-and-tables) compatible? Sorry if that's a stupid question :)

**Comment by andrewdwallo** [comment, marked as answer]
Yes, I'm pretty sure it is. Have you had any issues with using it?

**Reply by mirai-andrian** [reply]
I suppose it has nothing to do with dbal, will keep trying to make sense of it! Thanks for the reply!

**Reply by andrewdwallo** [reply]
Share your error message in the Filament Discord.
